src/main/java/com/metsci/laproc/App.java

```diff
@@ -19,12 +18,13 @@
 public class App {
 	public static void main( String[] args )
     {
-        BasicGraph graph = new BasicGraph(new Axis(0, 1, "X Axis"), new Axis(0, 1, "Y Axis"));
+        BasicGraph graph = new BasicGraph(new BasicAxis(0, 1, "X Axis"), new BasicAxis(0, 1, "Y Axis"));
```

Comment:
Can we name these based on our initial axis (false positive rate vs true positive rate)